test/node/replumb/require_node_test.cljs

```diff
@@ -359,14 +401,87 @@ trim-newline
+          out (unwrap-result res)]
+      (is (success? res) "(ns my.namespace (:require...])) and (foo.bar.core/add-five 30) should succeed")
+      (is (valid-eval-result? out) "(ns my.namespace (:require...])) and (foo.bar.core/add-five 30) should be a valid result")
+      (is (= "35" out) "(foo.bar.core/add-five 30) should be 35")
+      (repl/reset-env! '[my.namespace foo.bar.core foo.bar.macros]))
 
+    (let [res (do (read-eval-call "(ns my.namespace (:require [foo.bar.core :as f]))")
+                  (read-eval-call "(f/add-five 31)"))
+          out (unwrap-result res)]
+      (is (success? res) "(ns my.namespace (:require...:as...])) and (f/add-five 31) should succeed")
+      (is (valid-eval-result? out) "(ns my.namespace (:require...:as...])) and (f/add-five 31) should be a valid result")
+      (is (= "36" out) "(f/add-five 31) should be 36")
+      (repl/reset-env! '[my.namespace foo.bar.core foo.bar.macros])))
+
+  (deftest ns-macro-require-include-macros
+    ;; solved in 1.7.202 - see commented test below
+    (let [res (do (read-eval-call "(ns my.namespace (:require [foo.bar.baz :include-macros true]))")
+                  (read-eval-call "(foo.bar.baz/mul-baz 6 6)"))
+          error (unwrap-result res)]
+      (is (not (success? res)) "(ns my.namespace (:require ... :include-macros ...)) and (foo.bar.baz/mul-baz 6 6) should not succeed")
+      (is (valid-eval-error? error) "(ns my.namespace (:require ...:include-macros...)) and (foo.bar.baz/mul-baz 6 6) should be an instance of jsError")
+      (is (re-find #"Cannot read property 'call' of undefined" (extract-message error))
+          "(ns my.namespace (:require ...:include-macros...)) and (foo.bar.baz/mul-baz 6 6) should have correct error message.")
+      (repl/reset-env! '[my.namespace foo.bar.baz]))
+
+    ;; (let [res (do (read-eval-call "(ns my.namespace (:require [foo.bar.baz :include-macros true]))")
+    ;;               (read-eval-call "(foo.bar.baz/mul-baz 6 6)"))
+    ;;       out (unwrap-result res)]
+    ;;   (is (success? res) "(ns my.namespace (:require ... :include-macros ...)) and (f/mul-baz 6 6) should succeed")
+    ;;   (is (valid-eval-result? out) "(ns my.namespace (:require ...:include-macros...)) and (f/mul-baz 6 6) should be a valid result.")
+    ;;   (is (= "36" out) "(f/mul-baz 6 6) should be 36")
+    ;;   (repl/reset-env! '[my.namespace foo.bar.baz]))
+
+    (let [res (do (read-eval-call "(ns my.namespace (:require [foo.bar.core :as f :include-macros true]))")
+                  (read-eval-call "(f/mul-core 30 5)"))
+          out (unwrap-result res)]
+      (is (success? res) "(ns my.namespace (:require...:include-macros...])) and (f/mul-core 30 5) should succeed")
+      (is (valid-eval-result? out) "(ns my.namespace (:require...:as...])) and (f/mul-core 30 5) should be a valid result")
+      (is (= "150" out) "(f/mul-core 30 5) should be 150")
+      (repl/reset-env! '[my.namespace foo.bar.core foo.bar.macros])))
```

Comment:
The above works in `1.7.170` but not the previous.
In `1.7.202` they both work.